app/view-plants/page.tsx

```diff
@@ -82,6 +85,9 @@ export default function Page() {
   const [searchTerm, setSearchTerm] = useState<string>('');
   const [selectedPlants, setSelectedPlants] = useState<Plant[]>([]);
   const [ownedPlants, setOwnedPlants] = useState<OwnedPlant[]>([]);
+  const [isCardKeyOpen, setIsCardKeyOpen] = useState<boolean>(false);
```

Comment:
optimization note: each time `isCardKeyOpen` changes, the entire page will re-render, which is expensive and unnecessary, so we should consider 
1. maybe separate out the PlantCardList out into a separate component, or
2. wrap the PlantCardList section (i.e. everything under the title) in a useMemo? 

for more on re-rendering / how to prevent it: https://www.developerway.com/posts/react-re-renders-guide